lib/sc/Engine_SoftCut.sc

```diff
@@ -50,6 +50,7 @@ Engine_SoftCut : CroneEngine {
 		gr = Event.new;
 		gr.pb = Group.new(context.xg, addAction:\addToTail);
 		gr.rec = Group.after(context.ig);
+		gr.voices = ParGroup.new(context.xg, addAction:\addToHead);
```

Comment:
Wouldn't it be better if engines used the same naming where possible?
I.e. here we're assigning the `ParGroup` to `gr.voices`, but in PolySub the `ParGroup` is assigne to `gr`, which I found slightly confusing.

Reply:
ok